import_test.go

```diff
@@ -206,6 +206,22 @@ func TestImporter_Commit(t *testing.T) {
 	require.True(t, has)
 }
 
+func TestImporter_Commit_ForwardVersion(t *testing.T) {
+	tree, err := NewMutableTree(db.NewMemDB(), 0, false)
+	require.NoError(t, err)
+	importer, err := tree.Import(2)
```

Comment:
I don't really understand how we'd arrive here during a production import (import version ahead of a node in the stack). can you explain a bit more or provide an example?

Reply:
for example, if there are no tree operations (like set, delete) for the recent 2 versions, we have a root of version `n`, and need to import version `n+2`. In that case, we are only storing the root of version `n` at import previously and not storing `n+2`th root which is the previous version root link, so when commit the tree, it says there is no version `n+2` in the tree.

Reply:
did this change from the previous code? or was it handled by orphans?